### PR TITLE
added release notes for 0.18

### PR DESCRIPTION
**Pull request by puja108**
still waiting for update strategies

### CODE REVIEW

**Comment by puja108** [comment]
first shot at the release notes

@ewoutp @hectorj2f feel free to change and add to it, not sure I got all of it right. I might need someone to better explain --config-dir

File: content/reference/cli/release-notes.md [hunk 1]
```diff
@@ -10,6 +10,16 @@ weight = 10
 
 Here we list the more important changes we made between releases. See our [installation reference](/reference/cli/installation/) for information on how to update to the latest CLI.
 
+## Version 0.18.0
+
+Released 2015-06-19
+
+* There's a new component-level configuration key `pod`, with which you can group several components for sharing namespaces between them- You can read more about it on the [`swarm.json` reference page](/reference/cli/swarm-json#pod).
```
Comment (ewoutp) [inline content/reference/cli/release-notes.md]: `them-` -> `them.`

**Comment by ewoutp** [comment]
I'm fine with this. I don't think the config-dir is that important for normal users, so perhaps you can add that this is not needed/intended for normal use.

**Comment by marians** [comment]
We should set the release date and page lastmod date to June 22.

**Comment by puja108** [comment]
anything else?

**Pull request merged**